angular/projects/spark-angular/src/lib/components/sprk-tooltip/sprk-tooltip.stories.ts

```diff
@@ -0,0 +1,52 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { SprkTooltipModule } from './sprk-tooltip.module';
+import { SprkTooltipComponent } from './sprk-tooltip.component';
+import { markdownDocumentationLinkBuilder } from '../../../../../../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: 'Components/Tooltip',
+  component: SprkTooltipComponent,
+  decorators: [
+    storyWrapper(
+      storyContent => (
+        `<div class="sprk-o-Box">${storyContent}<div>`
+      )
+    )
+  ],
+  parameters: {
+    info: `${markdownDocumentationLinkBuilder('tooltip')}`,
+    docs: { iframeHeight: 160 },
+  },
+};
+
+const modules = {
+  imports: [
+    SprkTooltipModule,
+    BrowserAnimationsModule,
+  ],
+};
+
+export const defaultStory = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-tooltip
+      triggerIconType="question-filled"
```

Comment:
`triggerIconType` has the default set of 'question-filled' so we don't need this here. This adds unnecessary code for the user to copy when we already set it as the default.